### PR TITLE
Add scale out/in job

### PR DESCRIPTION
**Pull request by charlesakalugwu**
/cc @kargakis @mjudeikis @kwoodson

### CODE REVIEW

**Comment by kargakis** [comment]
Replicate my follow-up fixes too

https://github.com/openshift/release/pull/2238
https://github.com/openshift/release/pull/2239

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml [hunk 1]
```diff
@@ -247,6 +247,57 @@ periodics:
...
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 24h
+  name: periodic-ci-azure-scale-out-in-e2e
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --git-ref=charlesakalugwu/openshift-azure@scale-out-in-e2e-tests
```
Comment (charlesakalugwu) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]: Should be updated once openshift/openshift-azure#795 merges.

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml [hunk 2]
```diff
@@ -247,6 +247,57 @@ periodics:
     - configMap:
         name: prow-job-cluster-launch-e2e-azure
       name: job-definition
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 24h
+  name: periodic-ci-azure-scale-out-in-e2e
```
Comment (kargakis) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]: periodic-ci-azure-scaleup

Comment (kargakis) [reply ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]: don't feel strong though

**Comment by charlesakalugwu** [comment]
> Replicate my follow-up fixes too
> 
> #2238
> #2239

i've added those as well

**Pull request merged**